services/ui-src/src/components/pages/ReviewSubmit/McparReviewSubmitPage.tsx

```diff
@@ -54,38 +64,63 @@ export const McparReviewSubmitPage = () => {
   const submitForm = async () => {
     setSubmitting(true);
     if (isPermittedToSubmit) {
-      await submitReport(reportKeys);
+      const submissionDate = Date.now();
+      await submitReport(reportKeys, {
+        metadata: {
```

Comment:
These changes are made in submit report now, so we don't have to add this data here